dockerfiles/alpine/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM rust:1.72 as backend
+
+ENV HOME=/home/root
+WORKDIR $HOME/app
+# rust toolchain
+ARG RUST_TC_VER=stable
+ARG BUILD_TARGET=x86_64-unknown-linux-musl
+ARG BUILD_PROFILE=release-lto
```

Comment:
As I wrote in this comment: https://github.com/qoollo/bob-management/pull/9#discussion_r1335064566, you should pass commit hash and branch/tag name as an arguments here, because git repo is not accessible within the docker container